package/object_storage/object_storage.py

```diff
@@ -11,11 +13,39 @@ def __init__(self, PW, SWO):
         self.SWO = SWO
         self.PW = PW
         self.counter = 0
+        self.label_counter = 0
+        self.feature_flag = False
+
+    def save(self, path):
+        jsonString = json.dumps(self.storage)
+        jsonFile = open(path, "w")
+        jsonFile.write(jsonString) 
+        jsonFile.close()
+
+    def load(self, path):
+        with open(path, "r") as f:
+            objects = json.load(f)
+
+        self.feature_flag = False
+        for obj in objects.values():
+            self.create(obj)
+        self.feature_flag = True
+
+    def enable_intersections(self, intersections):
+        self.feature_flag = intersections
+        if self.feature_flag:
+            intersections = self.objManager.compute_intersections()
+            self.PW.drawIntersections(intersections)
```

Comment:
Have you spoken to @Fenik2305. Is his function deletes previous intersections if you call it with new one?